README.md

```diff
@@ -56,3 +56,39 @@ npm install -g webpack
 npm install
 ```
 
+## API
```

Comment:
nice job adding documentation for each of the routes; I don't know if this is standard, but for mine I also specified what the success / error codes would be

Reply:
Thanks! I should do that. Currently my error codes aren't very detailed, so I will fix it next PR and update this README with your suggestions. Currently it's only 200 and 400 responses, so it's not very informative.